src/pages/documentation/adobe-journey-optimizer/release-notes.md

```diff
@@ -6,10 +6,10 @@
 
 * Adds support for Adobe Journey Optimizer powered in-app messages.
 
-### Android AEPMessaging 2.0.1
+### Android Messaging 2.0.1
 
 * Adds support for Adobe Journey Optimizer powered in-app messages.
-* Adds support for AEP 2.x extensions. Note, AEPMessaging 2.0.1 is not backwards compatible with the ACP 1.x extensions.
+* Adds support for AEP 2.x extensions. Note, Messaging 2.0.1 is not backwards compatible with the ACP 1.x extensions.
```

Comment:
"Please note that Messaging 2.0.1 is not..."

"... with the ACP-prefixed 1.x extensions."

(Also, is it 2.0.1? It's not 2.0 that starts that?)

Reply:
... is not backwards compatible with MobileCore 1.x and compatible extensions. 
(same for the general notes)

Reply:
I have removed this line, as we don't mention it for other extensions. We have also covered this in other places.